libgadget/Makefile

```diff
@@ -150,3 +152,21 @@ config.h: Makefile $(CONFIG) $(INCL:config.h=)
 	mkdir -p `dirname $@`
 	MPICC="$(MPICC)" CFLAGS="$(CFLAGS)" OPT="$(OPT)" OPTIMIZE="$(OPTIMIZE)" VERSION="$(VERSION)" bash makeconfig.sh $@
 
+MPI_SHOWME := $(shell $(MPICC) -showme)
+
+.objs/%.compdb_entry: %.c
```

Comment:
Are these targets used anywhere?

It looks like a useful tool, but perhaps adding a comment about what it does somewhere, eventually?